src/wwwroot/controllers/RegistrationCtrl.js

```diff
@@ -33,7 +32,7 @@
         lastName: vm.lastName,
         password: vm.password,
         account_name: vm.accountName,
```

Comment:
Could you add a validation for accountName? it should only accept lowecase letters, numbers, `_` and `-`.